src/Sylius/Bundle/AttributeBundle/Resources/config/services.xml

```diff
@@ -45,6 +47,9 @@
             <argument type="service" id="sylius.registry.attribute_type" />
             <tag name="validator.constraint_validator" alias="sylius_valid_attribute_value_validator" />
         </service>
+
+        <service id="sylius.factory.attribute_value_type.configuration" class="%sylius.factory.attribute_value_type.configuration.class%">
```

Comment:
Instead of this two lines you should use `<service id="sylius.factory.attribute_value_type.configuration" class="%sylius.factory.attribute_value_type.configuration.class%" />` with **/** at the end.